.babelrc.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
   ],
   "plugins": [
     path.resolve(__dirname, './plugins/pbjsGlobals.js'),
-    useLocal('babel-plugin-transform-object-assign')
+    useLocal('babel-plugin-transform-object-assign'),
+    useLocal('@babel/plugin-proposal-object-rest-spread')
```

Comment:
Not related to using terser, however this fixes other build errors thrown by webpackStream if you target just Chrome. `@babel/plugin-proposal-object-rest-spread` is part of `babel/preset-env`.